boards/arm/imxrt/teensy-4.x/src/imxrt_flexspi_nor_boot.h

```diff
@@ -91,7 +91,7 @@
 
 /* Located in Destination Memory */
 
-#define IMAGE_ENTRY_ADDRESS        ((uint32_t)&_vectors)
+#define IMAGE_ENTRY_ADDRESS        ((uint32_t)_vectors)
```

Comment:
There is no issue in getting address of an array. Probably this is done to keep style in sync with other places

Reply:
Yes.